src/niweb/apps/noclook/views/create.py

```diff
@@ -10,19 +10,50 @@
 from django.contrib import messages
 from django.http import Http404
 from django.shortcuts import render, redirect
+from dynamic_preferences.registries import global_preferences_registry
 from apps.noclook import forms
 from apps.noclook.forms import common as common_forms
 from apps.noclook.models import NodeHandle, Dropdown
 from apps.noclook import helpers
 from apps.noclook import unique_ids
 from norduniclient.exceptions import UniqueNodeError, NoRelationshipPossible
 
-TYPES = [
-    ("contact", "Contact"),
-    ("organization", "Organization"),
-    ("group", "Group"),
-    ("role", "Role"),
-]
+
+global_preferences = global_preferences_registry.manager()
+menu_mode = global_preferences['general__menu_mode']
+
+if menu_mode == 'ni':
```

Comment:
Are all these changes meant to be in a separate parallel `sri` instance?
If that is the case then I think it should perhaps have been a separate app instead of being baked into noclook...

Reply:
The goal is to keep both NI data and SRI data in the same backend. So this is a first step.
That said, we can wait with this PR until we have the new frontend and graphql api done. Then these special cases can be removed.